src/Components/Chakra__TagCloseButton.res

```diff
@@ -409,6 +409,7 @@ let makeProps = (
   /**
     Component Props {TagCloseButton}
    **/
+   ~isDisabled=?,
```

Comment:
We can use `primitive types` for this (`bool`). You you need to changes type definition for `isDisabled`. 
```diff
-- isDisabled: Identity.t=?,
++ isDisabled: bool=?,
```

So, we can removed `~isDisabled=?` in `makeProps` and `~isDisabled=?isDisabled->Bool.fromOption,`

@fncolon

Reply:
Solved by 5a8809e8a68f69825efa078e9cda5431c932fd32 commit